### DISCUSSION TITLE
Property type not supported in Livewire!

### DISCUSSION THREAD

**Discussion by aoeng** [Laravel-Lang/common#156] [Q&A]
LocaleData to collect?
```
 $this->local = Locales::get(App::currentLocale());
```

**Comment by andrey-helldar** [comment]
The `Locales::get` method returns the `LocaleData` instance. If you want to use it, then you should explicitly get the locale code from the result:

```php
$this->local = Locales::get(App::currentLocale())->code;
// or
$this->local = Locales::raw()->get(App::currentLocale());
```

At the same time, if you need to take the current locale, you can use the `getDefault` method:

```php
$this->local = Locales::getDefault()->code;
// or
$this->local = Locales::raw()->getDefault();
```

If you are using version 2 of the `laravel-lang/locales` package, then you can use the `getCurrent` method, which is an alias for the `getDefault` method:

```php
$this->local = Locales::getCurrent()->code;
// or
$this->local = Locales::raw()->getCurrent();
```

On our website you can find more detailed information on using the `Locales` facade: https://laravel-lang.com/usage-facade.html
